jans-cedarling/cedarling/src/authz/mod.rs

```diff
@@ -259,6 +262,48 @@ impl Authz {
     }
 }
 
+/// Adds the following entities onto the context:
+///   - Jans::User
+///   - Jans::Workload
+///   - Jans::Access_token
+///   - Jans::id_token
+///   - Jans::Userinfo_token
+fn add_entities_to_context(
```

Comment:
For me more suitable name for this function is `build_context`.

Reply:
changed the name to `build_context` here: [c322173](https://github.com/JanssenProject/jans/pull/10275/commits/c322173167bab2f522b3011dc1ec62612b12c723)

Reply:
I see only comment update near function